plugin.video.themoviedb.helper/changelog.txt

```diff
@@ -0,0 +1,90 @@
+**1.0.3** 
```

Comment:
Please drop the changelog.txt in favor of the news tag in addon.xml. This file is no longer parsed by kodi.

Reply:
Is this mandatory? As the `<news>` tag is not intended to contain a complete history of changes, I feel it is not functionally equivalent to the a changelog (which serves a separate important purpose). I take the correct use of the `<news>` tag as specifying changes between official releases (which there are currently none as the plugin is not yet on the repo, hence why there is no "news").

Happy to drop the changelog if mandatory and only keep it locally, but I do feel that important details are lost through its removal.

Reply:
Well, there's no reason to include the file (which will tend to increase in size as updates go by) and make Kodi retrieve it from kodi's infrastructure if it is not used at all. I'd suggest you keep the changelog in the upstream repository and avoid including it in the distribution (this) repo. The news tag is only for the last version of the addon though.

Reply:
Fair enough. I've removed it from here and kept it in upstream.